reader_test.go

```diff
@@ -96,7 +96,40 @@ func TestDecodeMasterPlaylistWithAlternatives(t *testing.T) {
 			t.Fatal("should not be alternatives for this variant")
 		}
 	}
-	// fmt.Println(p.Encode().String())
+	//fmt.Println(p.Encode().String())
```

Comment:
This line probably didn't need to be changed. We can remove it in another cleanup.